src/modules/legal/assets/templates/aup.template.md

```
@@ -0,0 +1,22 @@
+# Acceptable Use Policy
+
+_Last updated: {{entity.lastUpdated}}_
+
+By using **{{entity.name}}** services, you agree to comply with this Acceptable Use Policy ("AUP").
+
+## 1. Prohibited content
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)